packages/kit/src/exports/node/index.js

```diff
@@ -95,6 +95,8 @@ function get_raw_body(req, body_size_limit) {
 /** @type {import('@sveltejs/kit/node').getRequest} */
 export async function getRequest({ request, base, bodySizeLimit }) {
 	return new Request(base + request.url, {
+		// @ts-expect-error
```

Comment:
I would expect we shouldn't need this. Is there some detail we can add here about how/when we might be able to remove it. Are the types provided by Typescript or Undici? Is there and open issue we can link to or can we file a new one?

Reply:
These types appear to be coming from TS, not from undici. Locally, even when I updated the Kit repo to undici 5.12.0, TS still complained about the `duplex`.